int32 motor
int32 angle     # between -150 and 150
int32 playtime  # between 100 and 250